speechbrain/core.py

```diff
@@ -1430,7 +1382,7 @@ def _fit_train(self, train_set, epoch, enable):
         steps_since_ckpt = 0
         with tqdm(
             train_set,
-            initial=self.step,
+            initial=self.optimizer_step,
```

Comment:
This is a pretty significant change to behavior IMO... I'm wondering if we want a separate issue to discuss this. For example, we might want it to be configurable or alternatively to explain somehow that the TQDM bar now iterates optimizer updates rather than batches. Basically I think this point needs more discussion because it seems a major change to me.

Reply:
Here are yet more possible approaches to consider:

- If grad_accumulation_factor is specified, add a second progress bar with the optimizer update progress
- If grad_accumulation_factor is specified, add an entry to the progress bar with `opt_ups=4/7` or whatever

Reply:
Thanks Peter! But I think I can just remove this for now. We could about it a bit better later on, but I don't see this as something critical for the next release?